src/overlays/actors/ovl_En_Syateki_Crow/z_en_syateki_crow.h

```diff
@@ -5,7 +5,8 @@
 
 #define EN_SYATEKI_CROW_GET_PARAM_F(thisx) ((thisx)->params & 0xF)
 #define EN_SYATEKI_CROW_GET_PARAM_F0(thisx) (((thisx)->params & 0xF0) >> 4)
-#define EN_SYATEKI_CROW_GET_PARAM_FF00(thisx) (((thisx)->params & 0xFF00) >> 8)
+#define EN_SYATEKI_CROW_GET_INDEX(thisx) (((thisx)->params & 0xFF00) >> 8)
```

Comment:
What kind of index do you mean?

Reply:
Each guay is given a unique index that identifies them. There are nine guays with indices from 0-8. These indices are used to toggle the correct `guayFlags` on and off. `sGuayFlagsPerWave` shows which flags are toggled for each wave; each flag corresponds to a guay with a particular index.

The same concept applies to all shooting gallery actors (Wolfos, Scrubs, etc).

Reply:
Hmm, Index just seems to broad (there are so many different kind of indices). Perhaps ID/MINIGAME_ID would be better, or clarify it is a flags index?

Reply:
Well, it's not used *just* for flags:
```c
    for (i = 0; i < EN_SYATEKI_CROW_GET_INDEX(&this->actor); i++) {
        path = &play->setupPathList[path->unk1];
    }
```
Not sure if that changes your thoughts on this.

Reply:
Interesting, I think `Id` still makes the most sense. Perhaps with the comment that the `Id` is used to index into different things?